snowshu/adapters/target_adapters/base_target_adapter.py

```diff
@@ -129,6 +133,18 @@ def load_data_into_relation(
 
         logger.info(final_message)
 
+    def clone_relation(self, relation: Relation) -> None:
+        """Clones a relation in the target.
+        Args:
+            relation: The relation to clone.
+        """
+        database_name = self.create_database_name(relation.database)
+        clone_query = f"""
+            CREATE TABLE IF NOT EXISTS {database_name}.{relation.schema}.{relation.name} AS
```

Comment:
Shouldn't this be create or replace?

Reply:
I'm gonna change it for now, as create or replace gonna give us always the latest subset, tho cannot think of scenario where it is going to have an advantage.

Reply:
I think in such scenario the query is ignored and process continues